pandas/tests/indexes/datetimes/test_datetime.py

```diff
@@ -794,3 +794,19 @@ def test_slice_duplicate_monotonic(self):
         result = idx._maybe_cast_slice_bound('2017-01-01', 'left', 'loc')
         expected = Timestamp('2017-01-01')
         assert result == expected
+
+    def test_split_datetime_index_with_timezone(self):
+        # https://github.com/pandas-dev/pandas/issues/14042
+        # check if timezone is correctly handled in array_wrapper
+        tz = 'Asia/Seoul'
+        index = date_range('2000-01-01', periods=10, freq='D', tz=tz)
+        series = Series(index, np.arange(10))
+        index_split = np.split(index, indices_or_sections=5)
```

Comment:
you are testing some extraneous things here, just pass ``indices_or_section=[]`` like in the original issue.